links/src/utils/getParent.ts

```diff
@@ -0,0 +1,26 @@
+import { ToTree, Tree, GetSiblings } from "./generateLinks";
+
+// getParents(tree, '/cancerqld/research') get parent_id of slug
+export const getParents: GetSiblings = (arr, slug) => {
```

Comment:
```suggestion
export const getParents: GetParents = (arr, slug) => {
```
This should use a GetParents type, not GetSiblings